R/logging.r

```diff
@@ -2,18 +2,29 @@
 # Per default, only error messages are shown
 # levels: 3 = DEBUG, 2 = INFO/MSG, 1 = ERROR
 
+#' A set of exported logging utilities that have the capability to be used in upstream projects
+#' Log level and log file can be set via R package options e.x. options(jupyter.log_level = 2L))
+#' or from the environment variables JUPYTER_LOG_LEVEL and JUPYTER_LOGFILE
+#'
+#' @name log_*
+#' @param \dots message to log
```

Comment:
please do `...`, i think `R cmd check` can’t interpret this.

also there need to be two spaces after the parameter name, i.e. `...  message to log`